### PR TITLE
docs(aio): fix TS warning error

### PR DESCRIPTION
**Pull request by albertopiras**
filter expects a boolean function param

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ x] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
filter expects a boolean function param

## Does this PR introduce a breaking change?
```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: aio/content/examples/rx-library/src/operators.2.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { filter, map } from 'rxjs/operators';
 
 const squareOdd = of(1, 2, 3, 4, 5)
   .pipe(
-    filter(n => n % 2),
+    filter(n => !!(n % 2)),
```
Comment (gkalpak) [inline aio/content/examples/rx-library/src/operators.2.ts]: Let's change it to `n % 2 !== 0` to make it more clear for beginners.

**Review by gkalpak — CHANGES_REQUESTED** [review]

File: aio/content/examples/rx-library/src/operators.2.ts (hunk 1 above)
Comment (albertopiras) [reply aio/content/examples/rx-library/src/operators.2.ts]: ok @gkalpak

**Review by gkalpak — APPROVED** [review]
Thx! Can you please squash the commits?

**Review by albertopiras — COMMENTED** [review]
Refactor: filter expects a boolean function param

**Comment by albertopiras** [comment]
@gkalpak ok, done

**Review by gkalpak — APPROVED** [review]

**Pull request closed without merging**